### DISCUSSION TITLE
Whatever I choose as owner side for OneToOne, migration will not care about it

### DISCUSSION THREAD

**Discussion by Loown** [mikro-orm/mikro-orm#4967] [General]
### Describe the bug

I'm trying to make a OneToOne relation, when I create migration with cli, foreign key is not given to the owner 

![Capture d’écran du 2023-11-26 19-19-47](https://github.com/mikro-orm/mikro-orm/assets/51403036/f8f8bcc2-d3d3-4684-bb1d-1affdc7983b1)
![Capture d’écran du 2023-11-26 19-19-31](https://github.com/mikro-orm/mikro-orm/assets/51403036/33af7875-176d-465b-8c0d-3b93200a4872)

phone_id is given to users table whatever I give it owner or not

PS: if there is a Discord community I'd love to join

### Reproduction

Should be enough

### What driver are you using?

@mikro-orm/mysql

### MikroORM version

5.9.4

### Node.js version

20

### Operating system

Ubuntu

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/mikro-orm/mikro-orm/blob/master/CONTRIBUTING.md).
- [X] Read the [docs](https://mikro-orm.io/docs).
- [X] Check that there isn't [already an issue](https://github.com/mikro-orm/mikro-orm/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mikro-orm/mikro-orm/discussions) or join our [Slack](https://join.slack.com/t/mikroorm/shared_invite/enQtNTM1ODYzMzM4MDk3LWM4ZDExMjU5ZDhmNjA2MmM3MWMwZmExNjhhNDdiYTMwNWM0MGY5ZTE3ZjkyZTMzOWExNDgyYmMzNDE1NDI5NjA).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by B4nan** [comment]
Not reproducible on my end, few things you can try:
- make sure your CLI is set up correctly to use TS files in dev mode
- build your project to rule out its not about outdated dist folder (but if your project is set up correctly, and you have ts-node installed, TS files should be preferred)
- remove migration snapshot file
- remove the 1:1 decorator mapped by option from the owning side (e.g. `@OneToOne({ entity: () => User, nullable: true })`

**Reply by Loown** [reply]
Works with removing the 1:1 decorator mapped by + forcing owner: false in the other side (should be documented maybe)

**Reply by B4nan** [reply]
Well, I won't document something I can't reproduce, there are many tests for this too. My bet is the snapshot file.
